### PR TITLE
MAILBOX-310 Extract ACL diff from ACLChanged

### PR DESCRIPTION
**Pull request by aduprat**
Only last commit

### CODE REVIEW

**Comment by chibenwa** [comment]
```
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   CassandraSetMessagesMethodTest>SetMessagesMethodTest.attachmentsShouldBeRetrievedWhenChainingSetMessagesAndGetMessagesTextAttachment:2985 » ConditionTimeout
[INFO] 
[ERROR] Tests run: 997, Failures: 0, Errors: 1, Skipped: 1
```

Maybe just unlucky.

I will schedule the build after the **master** build.

**Comment by quynhn** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

File: server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserMailboxesRoutesTest.java [hunk 1]
```diff
@@ -349,7 +349,8 @@ public void putShouldReturnOkWhenIssuedTwoTimes() {
         @Test
         public void putShouldAddAMailbox() {
             with()
-                .put(MAILBOX_NAME);
+                .put(MAILBOX_NAME)
+            .prettyPeek();
```
Comment (rouazana) [inline server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserMailboxesRoutesTest.java]: syso?

File: mailbox/api/src/main/java/org/apache/james/mailbox/acl/ACLDiff.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+ * under the License.                                           *
+ ****************************************************************/
+package org.apache.james.mailbox.acl;
+
+import java.util.Map;
+import java.util.stream.Stream;
+
+import org.apache.james.mailbox.model.MailboxACL;
+
+public class ACLDiff {
```
Comment (rouazana) [inline mailbox/api/src/main/java/org/apache/james/mailbox/acl/ACLDiff.java]: PositiveUserACLDiff?

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
https://github.com/linagora/james-project/pull/1045 for testing

**Pull request closed without merging**